### ISSUE TITLE
Should ignore `declare module`

### ISSUE THREAD

**Issue by andreialecu** [edvardchen/eslint-plugin-i18next#41]
It seems the plugin currently flags this as an error:

```ts
declare module 'country-emoji' {
  export function flag(code: string): string;
}
```

```
module "country-emoji"
disallow literal stringeslint(i18next/no-literal-string)
```

**Comment by edvardchen** [comment]
U R right. Let me fix
